### PR TITLE
#36 Remove Fault Clearing Logic

### LINKED ISSUE #36: Remove Motor Controller faulting reset logic
This is probably contained within the TSMS class/in the cascadiaMC class, this logic is now handled entirely in hardware

### PR DESCRIPTION
**Pull request by Peyton-McKee**
Removed logic from mpu for clearing faults automatically. 

closes #36

### CODE REVIEW

**Review by nwdepatie — CHANGES_REQUESTED** [review]
I think there's some more to be done, not sure if this would work when uploaded to the car

**Review by nwdepatie — APPROVED** [review]
Jk disregard my previous thingy, I think this will work

**Comment by nwdepatie** [comment]
Pls verify on hardware before merging tho

File: MPU/src/cascadiamc.cpp [hunk 1]
```diff
@@ -92,17 +92,6 @@ uint16_t CascadiaMC::getTorque()
...
 
 
-void CascadiaMC::clearFault()
-{
-    int time = millis() + 250;
-    while(millis() < time) {
-        sendMessageCAN1(CANMSG_MC_SETPARAMETER, 8, FAULT_CLEAR);
-        delay(5);
-    }
-    isFaulted = false;
```
Comment (mccauleyma) [inline MPU/src/cascadiamc.cpp:102]: This "isFaulted" variable is not addressed elsewhere?

**Review by mccauleyma — CHANGES_REQUESTED** [review]

File: MPU/src/cascadiamc.cpp (hunk 1 above)
Comment (Peyton-McKee) [reply MPU/src/cascadiamc.cpp:102]: it is, but it will get passed when we clear the fault manually i believe

Comment (mccauleyma) [reply MPU/src/cascadiamc.cpp:102]: Passed?

Comment (Peyton-McKee) [reply MPU/src/cascadiamc.cpp:102]: like set to false

Comment (Peyton-McKee) [reply MPU/src/cascadiamc.cpp:102]: so like this automatically did it, but when we press the button the clear fault thing will do it itself

Comment (mccauleyma) [reply MPU/src/cascadiamc.cpp:102]: I'm worried we may be missing a step here, but we can diagnose tomorrow on the actual car

**Comment by nwdepatie** [comment]
Closed because we added it to the pdu integration to get the car working

**Pull request closed without merging**